src/server.cpp

```diff
@@ -2863,28 +2863,28 @@ std::wstring Server::handleChat(const std::string &name, const std::wstring &wna
 {
 	// If something goes wrong, this player is to blame
 	RollbackScopeActor rollback_scope(m_rollback,
-		std::string("player:") + name);
+			std::string("player:") + name);
 
 	if (g_settings->getBool("strip_color_codes"))
 		wmessage = unescape_enriched(wmessage);
 
 	if (player) {
 		switch (player->canSendChatMessage()) {
-			case RPLAYER_CHATRESULT_FLOODING: {
-				std::wstringstream ws;
-				ws << L"You cannot send more messages. You are limited to "
-				   << g_settings->getFloat("chat_message_limit_per_10sec")
-				   << L" messages per 10 seconds.";
-				return ws.str();
-			}
-			case RPLAYER_CHATRESULT_KICK:
-				DenyAccess_Legacy(player->getPeerId(),
-						L"You have been kicked due to message flooding.");
-				return L"";
-			case RPLAYER_CHATRESULT_OK:
-				break;
-			default:
-				FATAL_ERROR("Unhandled chat filtering result found.");
+		case RPLAYER_CHATRESULT_FLOODING: {
+			std::wstringstream ws;
+			ws << L"You cannot send more messages. You are limited to "
+					<< g_settings->getFloat("chat_message_limit_per_10sec")
+					<< L" messages per 10 seconds.";
```

Comment:
1 indent is enough

Reply:
The code style allows that and LINT would probably have issues with any other notation due to its limited configuration set.

Reply:
@paramat Our code style demands double-tab indents for continuation lines, IIRC.

Reply:
Sorry, as long as it's readable any amount of indentation is ok.

Fuck what LINT thinks, 95% of it's requests are ridiculous =)
We're not going to make a mess of MT code just because LINT is crap.

> Our code style demands double-tab indents for continuation lines, IIRC.

I hope not, if it does that needs editing, 1 is fine except when the indented line would be aligned with a following indented line, like in a conditional.